pkg/system/phase1_verifying.go

```diff
@@ -171,3 +201,60 @@ func (r *Reconciler) CheckJoinSecret() error {
 	}
 	return nil
 }
+
+func (r *Reconciler) CheckExternalPg(postgresDbURL string) error {
+	if r.ExternalPgSecret.StringData["db_url"] == "" {
+		return util.NewPersistentError("InvalidExternalPgSecret",
+			"ExternalPgSecret is missing db_url")
+	}
+	db_url := r.ExternalPgSecret.StringData["db_url"]
+	if r.NooBaa.Spec.ExternalPgSSLRequired {
+		if !r.NooBaa.Spec.ExternalPgSSLUnauthorized {
+			db_url += "?sslmode=verify-full" // won't allow self-signed certs
+		} // when we want to allow self-signed we will use the default sslmode=require
+	} else {
+		db_url += "?sslmode=disable" // don't use ssl - the default is to use it
+	}
+	db, err := sql.Open("postgres", db_url)
+	if err != nil {
+		return util.NewPersistentError("InvalidExternalPgUrl",
+			fmt.Sprintf("failed openning a connection to external DB url: %q, error: %s",
+				db_url, err))
+	}
+	defer db.Close()
+	err = db.Ping()
+	if err != nil {
+		return util.NewPersistentError("InvalidExternalPgUrl",
+			fmt.Sprintf("failed pinging external DB url: %q, error: %s",
+				db_url, err))
+	}
+	// Query the PostgreSQL version
+	var version string
+	err = db.QueryRow("SELECT current_setting('server_version_num')::integer / 10000").Scan(&version)
+	if err != nil {
+		return util.NewPersistentError("InvalidExternalPgVersion",
+			fmt.Sprintf("failed getting version of external DB url: %q, error: %s",
+				db_url, err))
+	}
+	// Check if the version is 15
+	if version != "15" {
```

Comment:
Why do we want to limit it only to this specific version? 
Can we use a min version (maybe 12?)